client/api/CfClient.cs

```diff
@@ -46,6 +46,20 @@ public sealed class CfClient
         private static readonly Lazy<CfClient> lazy = new Lazy<CfClient>(() => new CfClient()); 
         public static CfClient Instance { get { return lazy.Value; }  }
 
+
+        [Obsolete("Method deprecated. Use Instance and Initialize instead ", false)]
+        public static async Task<CfClient> getInstance(string apiKey, Config config)
+        {
+            await CfClient.Instance.Initialize(apiKey, config);
+
+            return CfClient.Instance;
+        }
+        [Obsolete("Method deprecated. Use Initialize instead ", false)]
+        public static CfClient getInstance()
+        {
+            return CfClient.Instance;
+        }
+ 
         /// <summary>
```

Comment:
Revert old methods getInstance() and mark them as obsolete. They should have same behavior as old methods.